pages/spaces/dialogDemo.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import Button from "@material-ui/core/Button";
+import Avatar from "@material-ui/core/Avatar";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemAvatar from "@material-ui/core/ListItemAvatar";
+import ListItemText from "@material-ui/core/ListItemText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Dialog from "@material-ui/core/Dialog";
+import PersonIcon from "@material-ui/icons/Person";
+import AddIcon from "@material-ui/icons/Add";
+import Typography from "@material-ui/core/Typography";
+
+import {
+  useTheme,
+  createMuiTheme,
+  ThemeProvider,
+} from "@material-ui/core/styles";
+import { Container, Grid, Box } from "@material-ui/core";
+import SpacesCard from "../../components/SpacesCard";
+
+
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      dark: "#614885",
+      main: "#977BBF",
+      light: "#F5F2F9",
+    },
+    secondary: {
+      main: "#f44336",
+    },
+  },
+});
+   const friends = [
+     { name: "Charles Liu", status: "Finish work!" },
+     { name: "Charles Liu", status: "in study session!" },
+     { name: "Charles Liu", status: "taking a break" },
+   ];
+   const participants = [
+     { name: "Charles Liu", status: "Finish work!" },
+     { name: "Charles Liu", status: "in study session!" },
+     { name: "Charles Liu", status: "taking a break" },
+   ];
+   const hosts = [
+     { name: "Charles Liu", status: "Finish work!" },
+     { name: "Charles Liu", status: "in study session!" },
+     { name: "Charles Liu", status: "taking a break" },
+     { name: "Charles Liu", status: "Finish work!" },
+     { name: "Charles Liu", status: "in study session!" },
+     { name: "Charles Liu", status: "taking a break" },
+   ];
```

Comment:
look into nextjs default props please. It would be cleaner if you added placeholder information there. Not sure if documentation for `defaultProps` is still supported though. If you can't figure it out, resort to static props as a last resort